src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('tbody');
+const addRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const addColumButton = document.querySelector('.append-column');
+const removeColumButton = document.querySelector('.remove-column');
+const maxCount = 10;
+const minCount = 2;
+
+addRowButton.addEventListener('click', e => {
+  removeRowButton.disabled = false;
+
+  table.prepend(table.lastElementChild.cloneNode(true));
+
+  if (table.children.length >= maxCount) {
+    addRowButton.disabled = true;
+  }
+});
+
+removeRowButton.addEventListener('click', e => {
+  addRowButton.disabled = false;
+
+  table.lastElementChild.remove();
+
+  if (table.children.length <= minCount) {
+    removeRowButton.disabled = true;
+  }
+});
+
+removeColumButton.addEventListener('click', e => {
+  const allCell = document.querySelectorAll('tr');
+
+  allCell.forEach((item) => {
+    item.lastElementChild.remove();
+
+    minMax(item);
+  });
+});
+
+addColumButton.addEventListener('click', e => {
+  const allCell = document.querySelectorAll('tr');
+
+  allCell.forEach((item) => {
+    const td = document.createElement('td');
+
+    item.append(td);
+
+    minMax(item);
+  });
+});
+
+function minMax(item) {
```

Comment:
Please rename your function according to these rules. The name must be descriptive for other developers
![image](https://user-images.githubusercontent.com/96409563/208387408-eb0fb653-51a3-4d78-9489-e3dedb72bb30.png)

Reply:
done